src/server/scripts/EasternKingdoms/BlackrockMountain/MoltenCore/boss_baron_geddon.cpp

```diff
@@ -174,14 +174,13 @@ class spell_geddon_inferno : public SpellScriptLoader
                         break;
                     case 4:
                     case 5:
-                        multiplier = 3;
+                        multiplier = 4;
                         break;
                     case 6:
-                    case 7:
-                        multiplier = 4;
+                        multiplier = 6;
                         break;
-                    case 8:
```

Comment:
make this one still case 8 instead case 7 what you did. because tick 8 has to do 5000 damage